Lesson7/organizer7/scheduler/models.py

```diff
@@ -0,0 +1,22 @@
+# scheduler/models.py
+from django.db import models
+
+
+class User(models.Model):
```

Comment:
By using name `User' for the client of your service you are shading the default Django User model.

The simplest way to fix it just rename your `User` model to any other name e.g. `MyUser`

With that change, L6 could be removed.